test/export/data/country_codes_test.rb

```diff
@@ -0,0 +1,19 @@
+# encoding: utf-8
+
+require File.expand_path(File.join(File.dirname(__FILE__) + '/../../test_helper'))
+
+class TestCldrDataCountryCodes < Test::Unit::TestCase
+  test 'country codes' do
+    expected = [
+      {
+        "three_digit_code" => "958",
+        "three_letter_code" => "AAA",
+      },
+      {
+        "three_letter_code" => "ASC",
+      }
+    ]
+    country_codes = Cldr::Export::Data::CountryCodes.new
+    assert_equal(expected, [country_codes[:CountryCodes]["AA"], country_codes[:CountryCodes]["AC"]])
```

Comment:
Feels weird to be constructing an array for the "actual" side of the assert, using two asserts, or `assert_includes` would probably be more readable here.

Reply:
Good point... Adjusted